src/appengine/handlers/jobs.py

```diff
@@ -44,6 +57,37 @@ def get_queues():
   return queues
 
 
+def get_results(this):
+  """Get results for the jobs page."""
+  params = {k: v for k, v in this.request.iterparams()}  # pylint: disable=unnecessary-comprehension
+
+  query = datastore_query.Query(data_types.Job)
+
+  # Results sorted alphabatically by name.
+  # is_desc reverses the sorting.
+  query.order('name', is_desc=False)
+
+  page = helpers.cast(
+      this.request.get('page') or 1, int, "'page' is not an int.")
+
+  # Adding Filters for Search Query
+  filters.add(query, params, FILTERS)
```

Comment:
Let's group all the these lines like so to reduce vertical whitespace.

```python
# Return jobs sorted alphabetically by name
query = datastore_query.Query(data_types.Job)
query.order('name', is_desc=False)
filters.add(query, params, FILTERS)

page = helpers.cast(
      this.request.get('page') or 1, int, "'page' is not an int.")
items, total_pages, total_items, has_more = query.fetch_page(
      page=page, page_size=PAGE_SIZE, projection=None, more_limit=MORE_LIMIT)
```

Reply:
sure, thanks!